Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/FareCalculator.hs

```diff
@@ -264,8 +266,8 @@ data CalculateFareParametersParams = CalculateFareParametersParams
     distanceUnit :: DistanceUnit
   }
 
-calculateFareParameters :: MonadFlow m => CalculateFareParametersParams -> m FareParameters
-calculateFareParameters params = do
+calculateFareParameters :: MonadFlow m => Maybe (Id DMOC.MerchantOperatingCity) -> CalculateFareParametersParams -> m FareParameters
```

Comment:
you can add `merchantOperatingCityId` to `CalculateFareParametersParams` data type